integration/skaffold/helper.go

```diff
@@ -256,8 +276,14 @@ func (b *RunBuilder) RunOrFailOutput(t *testing.T) []byte {
 		}
 		t.Fatalf("skaffold %s: %v, %s", b.command, err, out)
 	}
+	//Create human readable time string
+	showsTime := humanize.Time(start)
 
-	logrus.Infoln("Ran in", time.Since(start))
+	//Case for when it takes less than a second
+	if time.Since(start).Seconds() < 1 {
+		showsTime = time.Since(start).String() + " ago"
+	}
+	logrus.Infoln("Ran", showsTime)
```

Comment:
this would make an excellent utility function so you don't have to repeat the code. 
```suggestion
	logrus.Infoln("Ran", util.showHumanizeTime(start))
```
and

```
def showHumanizeTime(start time.Time) string {

showsTime := humanize.Time(start)
if time.Since(start).Seconds() < 1 {
	return  time.Since(start).String() + " ago"
}
return humanize.Time(start)
}
```

Reply:
I've added a function in util and reworked its usage in the files.